Libraries/xcdriver/Headers/xcdriver/Usage.h

```diff
@@ -0,0 +1,21 @@
+/**
+ Copyright (c) 2016-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#ifndef __xcdriver_Usage_h
+#define __xcdriver_Usage_h
+
+#include <xcdriver/Base.h>
+
+namespace xcdriver {
+
+std::string usage();
```

Comment:
This works, but in here the more common convention is to put this in a class as a static method:

    class Usage {
    private:
        Usage();
        ~Usage();
    
    public:
        static std::string Text();
    };

The `private:` block is to make the class purely static — since the constructor is private, there's no way to create an instance of it. That's all good, though, since the only method on it is `Usage::Text()` and it's static.